notifier.rst

```diff
@@ -112,7 +113,7 @@ Service             Package                                DSN
 
 .. versionadded:: 6.3
 
-    The Bandwith, iSendPro, Plivo, RingCentral, SimpleTextin and Termii integrations
+    The Bandwith, iSendPro, Plivo, RingCentral, SimpleTextin, Termii and Redlink integrations
     were introduced in Symfony 6.3.
```

Comment:
Please create a new block for versionadded:: 6.4, because 6.3 is already in feature freeze, thanks

Reply:
Done

Reply:
Done